src/alias.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+use Interop\Http\Server\RequestHandlerInterface;
+use Psr\Http\Server\RequestHandlerInterface as PsrRequestHandlerInterface;
+
+class_alias(PsrRequestHandlerInterface::class, RequestHandlerInterface::class);
```

Comment:
Hm... before it was the other way - non-psr, psr, why it's changed now?

Reply:
Because the first argument of `class_alias` is the original class, that must exists and the second is the alias. If `http-interop/htttp-server-handler` is removed, it does not exists any more, so must be the alias, not the original class.

Reply:
So the example you've provided before is not working anymore:

```
Warning: Cannot declare interface Psr\Http\Server\RequestHandlerInterface, because the name is already in use in ~/test-http-middleware/test.php on line 11
```

Reply:
oh, wait ;) sorry, forgot to remove there `class_alias`, all is fine now !

Reply:
The alias is now autoloaded by composer so the previous code is not valid.
The same code without defining the alias should work fine:

```php
require 'vendor/autoload.php';

use Interop\Http\Server\RequestHandlerInterface;
use Interop\Http\Server\MiddlewareInterface;
use Psr\Http\Server\MiddlewareInterface as PsrMiddewareInterface;
use Psr\Http\Server\RequestHandlerInterface as PsrRequestHandlerInterface;
use Psr\Http\Message\ResponseInterface;
use Psr\Http\Message\ServerRequestInterface;

class HandlerBar implements RequestHandlerInterface
{
    public function handle(ServerRequestInterface $request): ResponseInterface
    {
    }
}

class HandlerFoo implements PsrRequestHandlerInterface
{
    public function handle(ServerRequestInterface $request): ResponseInterface
    {
    }
}

class Bar implements MiddlewareInterface
{
    public function process(ServerRequestInterface $request, RequestHandlerInterface $handler): ResponseInterface
    {
        return $handler->handle($request);
    }
}

class Foo implements PsrMiddewareInterface
{
    public function process(ServerRequestInterface $request, RequestHandlerInterface $handler): ResponseInterface
    {
        return $handler->handle($request);
    }
}
```

Reply:
checked also with zend-startigility dev and it seems to be fine !